### ISSUE TITLE
Cannot change chunks render in game (world, server) [BUG]

### ISSUE THREAD

**Issue by Tarquitet** [Armandukx/IdleFPS#2]
**Describe the bug**
If I have 8 (or another number) in chunk render, when I need to change to a different number, it didn't save the change with the other number.

**To Reproduce**
1. Go to 'graphics'
2. Click on 'chunk render'
3. Scroll down to 'different number'
4. Save it
5. See error??

**Expected behavior**
Well, If you need to change the render distance of chunks, obviously you need to change it.

**Screenshots**
I recorded a video to show the problem (Honestly. I don't know if that's something normal with the mod).
Link: https://youtu.be/HGoeAS_f1Uc

**Desktop (please complete the following information):**
 - OS: Windows 10.

**IdleFPS (please complete the following information):**
 - What mod version you were using: 1.0.7 version.
 - What game version: 1.19.4-fabric 0.14.21.
 
 (Psd: Sorry for my english 😥)

**Comment by Armandukx** [comment]
Have you tried changing the render distance without the mod that modifies the 'Video Settings' menu?

(Sorry for the late response)

**Comment by Tarquitet** [comment]
Of course, and I even installed the mod in a separate instance to test if it's some additional mod causing that error.
Of the mods that I tried, they have been Sodium, Vulkan, the others do not affect it or at least that is what I have seen in my tests (the mod + chunk rendering works perfectly without these mods mentioned above).
The Canvas renderer mod does not affect it.

**Comment by Armandukx** [comment]
So the bug is that, those mods have seperate 'Video Settings' names, as in Idle Tweaks cannot detect those 'Video Settings'.
It can only detect the default Minecraft 'Video Settings' or 'Video Settings' that havent been renamed to something else.

**Solution**
The solution would be for me to add support to those mods.

**Comment by Tarquitet** [comment]
Then. Will you release a new version of the mod with those supports?

**Comment by Armandukx** [comment]
It depends, I'll check it out if there is a way of detecting those settings, or I'll just have to make a better saving system.

**Comment by andreasdc** [comment]
That is a real bug

**Comment by MASUYAMA-github** [comment]
me too!

**Comment by GhostOps77** [comment]
so, is this issue even gonna be resolved or not?

**Comment by R0gyman** [comment]
this caused me hours of sorting through my mods to find out what was causing it please fix it :( +1

**Comment by andreasdc** [comment]
> this caused me hours of sorting through my mods to find out what was causing it please fix it :( +1

Just use DynamicFPS

**Comment by R0gyman** [comment]
> > this caused me hours of sorting through my mods to find out what was causing it please fix it :( +1
> 
> Just use DynamicFPS

I thought this was an addon to dynamic fps, mod shouldn't be out if you cant change your render distance its like the most anti-performance performance mod

**Comment by andreasdc** [comment]
> > > this caused me hours of sorting through my mods to find out what was causing it please fix it :( +1
> > 
> > 
> > Just use DynamicFPS
> 
> I thought this was an addon to dynamic fps, mod shouldn't be out if you cant change your render distance its like the most anti-performance performance mod

Nope, just don't use it and use DynamicFPS :)
